worker/src/bin/worker.rs

```diff
@@ -110,24 +103,34 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let node_name = std::env::var("NODE_NAME")
         .unwrap_or_else(|_| gethostname::gethostname().to_str().unwrap().to_owned());
 
+    // Run it
+    let addr = SocketAddr::new(args.listen_address, args.port.unwrap_or(0));
+    let listener = tokio::net::TcpListener::bind(addr).await?;
+    let addr = listener.local_addr()?;
+    log::info!("listening on {}", addr);
```

Comment:
Aren't we already printing the URL further down?

Reply:
```
$ grep -R 'log::info' worker/src/bin
worker/src/bin/worker.rs:    log::info!("listening on {}", addr);
worker/src/bin/worker.rs:        log::info!("Diagnostics available at {}", my_url);
worker/src/bin/worker.rs:                    log::info!("queue is empty");
worker/src/bin/dist.rs:            log::info!("Dist tarball {} created successfully", t.to_str().unwrap());
```

Don't think so?

Reply:
The second line

Reply:
`listening on` != `Diagnostics available`